tests/visualization_tests/test_edf.py

```diff
@@ -11,6 +14,14 @@ def test_target_is_none_and_study_is_multi_obj() -> None:
         plot_edf(study)
 
 
+def _validate(y_values: Sequence[float]) -> None:
```

Comment:
`_validate` sounds unclearn. how about `_validate_y_values`?
```suggestion
def _validate_y_values(y_values: Sequence[float]) -> None:
```